workflow-templates/generate-translation-files.yml

```diff
@@ -0,0 +1,13 @@
+# Run generate-translation-files
+
+name: Generate Translation Files
+
+on:
+  push:
+    branches:
+    - master
+  workflow_dispatch: {}
```

Comment:
(curious) I can't find anything on this syntax. does this just mean you don't have any params to `workflow_dispatch`? if so, could you just omit it?

Reply:
Makes sense. I wonder why I couldn't find documentation explaining that meaning.

Reply:
I like `workflow_dispatch:` better than `workflow_dispatch: {}`, esp since the curly braces are one of those "multiple ways to do it in YAML" things that you hardly ever see.